backend/wayland/seat.c

```diff
@@ -282,12 +283,40 @@ static struct wlr_wl_input_device *get_wl_input_device_from_input_device(
 	return (struct wlr_wl_input_device *)wlr_dev;
 }
 
+void create_wl_seat(struct wl_seat *wl_seat, struct wlr_wl_backend *wl) {
```

Comment:
`create_*` functions should allocate a new structure and return it.

OTOH, `init_*` functions fill an already-existing structure.

Reply:
This matches `create_wl_pointer` and `create_wl_keyboard` that do not return created objects. Though they link them in `backend->devices`.  
Similar to those this one takes care of "linking" in backend. It just happens that in this PR it is not behind `wl_list` in `backend`.

Note that in https://github.com/swaywm/wlroots/pull/2417 this method actually going to create objects and link it in `backend`. But I'm not fully sure it is correct to write `init_*` because we are not initializing single field.

I think `add_wl_seat` will be a better fit here.

Reply:
Name it `init` in this PR, rename to `create` in your other PR.

Reply:
As per suggestion in IRC, now `wlr_wl_seat` gets allocated independently from `wlr_wl_backend` and thus name `create_*` is applicable.